src/components/navigation/Header.tsx

```diff
@@ -29,45 +34,6 @@ function Header(header: NavProps): JSX.Element {
     }
   }, [pageProps]);
 
```

Comment:
To make the header component more easy to read I moved getNav function out side of the hook. I didn't move It outside the file because we probably don't need this outside of navigation.